packs/pathfinder-monster-core/pleroma.json

```diff
@@ -1758,7 +1758,7 @@
+                    "value": "<p><strong>Requirements</strong> The pleroma has a sphere of creation or sphere of oblivion active</p>\n<hr />\n<p><strong>Effect</strong> The pleroma makes one of its spheres fly 10 feet in any direction, ignoring difficult terrain and greater difficult terrain. A sphere of creation creates new matter in its path, which the pleroma can have manifest as normal terrain, difficult terrain, greater difficult terrain, or a cube of solid mater (such as clay, wood, or stone). A sphere of oblivion destroys unattended objects it touches, though larger objects are destroyed at a rate of one 10-foot cube per round of contact. The sphere can enter the space of a creature; when it does, the creature takes 20d6 damage with a @Check[fortitude|dc:43] save.</p>\n<p>This is an incapacitation effect.</p>\n<p><strong>Success</strong> The creature takes no damage and is pushed out of the sphere to the nearest open space of the GM's choice.</p>\n<p><strong>Failure</strong> The creature takes full damage; this is @Damage[20d6[vitality]] damage for a sphere of creation or @Damage[20d6[void]] damage for a sphere of oblivion, but it can damage any type of creature regardless of its normal immunities. The creature is then pushed out of the sphere as on a success. A creature reduced to 0 HP is slain instead of being pushed out, either merged with new matter for a sphere of creation or completely destroyed for a void of oblivion; the creature can be restored only via a @UUID[Compendium.pf2e.spells-srd.Item.Wish] ritual or similarly powerful effect. This is a death effect.</p>\n<p><strong>Critical Failure</strong> As failure, but the creature takes double damage.</p>"
```

Comment:
```suggestion
                    "value": "<p><strong>Requirements</strong> The pleroma has a sphere of creation or sphere of oblivion active</p>\n<hr />\n<p><strong>Effect</strong> The pleroma makes one of its spheres fly 10 feet in any direction, ignoring difficult terrain and greater difficult terrain. A sphere of creation creates new matter in its path, which the pleroma can have manifest as normal terrain, difficult terrain, greater difficult terrain, or a cube of solid mater (such as clay, wood, or stone). A sphere of oblivion destroys unattended objects it touches, though larger objects are destroyed at a rate of one 10-foot cube per round of contact. The sphere can enter the space of a creature; when it does, the creature takes 20d6 damage with a @Check[fortitude|dc:43] save.</p>\n<p>This is an incapacitation effect.</p>\n<p><strong>Success</strong> The creature takes no damage and is pushed out of the sphere to the nearest open space of the GM's choice.</p>\n<p><strong>Failure</strong> The creature takes full damage; this is @Damage[20d6[vitality]]{vitality} damage for a sphere of creation or @Damage[20d6[void]]{void} damage for a sphere of oblivion, but it can damage any type of creature regardless of its normal immunities. The creature is then pushed out of the sphere as on a success. A creature reduced to 0 HP is slain instead of being pushed out, either merged with new matter for a sphere of creation or completely destroyed for a void of oblivion; the creature can be restored only via a @UUID[Compendium.pf2e.spells-srd.Item.Wish] ritual or similarly powerful effect. This is a death effect.</p>\n<p><strong>Critical Failure</strong> As failure, but the creature takes double damage.</p>"
```